docs/dashboard.md

```diff
@@ -0,0 +1,33 @@
+# SC4S Metrics and Events Dashboard
+SC4S Metrics and Events Dashboard enables monitoring crucial metrics and events flow of all the SC4S instances sending data to a chosen Splunk platform.
+
+
+## Functionalities
+
+### Overview metrics
+The dashboard displays the cumulative sum of received and dropped messages of all SC4S instances in chosen interval and for the specified time range. It is 30 seconds span in a 15 minutes window by default.
```

Comment:
The dashboard displays the cumulative sum of received and dropped messages for all SC4S instances in a chosen interval and for the specified time range. By default the interval is set to 30 seconds and the time range is 15 minutes.